lua/KoalaVim/plugins/coding.lua

```diff
@@ -7,9 +7,8 @@ table.insert(M, {
 	'numToStr/Comment.nvim',
 	event = { 'BufReadPre', 'BufNewFile' },
 	config = function()
-		require('Comment').setup {
-
-			pre_hook = require('ts_context_commentstring.integrations.comment_nvim').create_pre_hook(),
+		require('Comment').setup{
```

Comment:
@ofirgall I was unable to convert this to opts.
From what I understand - it's because the pre_hook requires something that is granted by the dependencies, but is resolved before the dependencies when it's in a variable and not in the config function.

Reply:
I resolved the issue at d76ac2ceb0265db2bbeea13815e76adfc610820e.

Note: currently the user won't be able to disable the `pre_hook` but we would handle it as part of #16